spec/lib/iiif_manifest/v3/manifest_builder/canvas_builder_spec.rb

```diff
@@ -273,6 +276,29 @@ def id
       end
     end
 
+    context 'when record has see_also' do
+      before do
+        class MyWork
+          def id
+            'test-22'
+          end
+
+          def see_also
+            [{
+              id: 'test-22.json',
+              label: 'test-22 see also'
+            }]
+          end
+        end
+      end
+      it 'generates canvas without seeAlso property' do
```

Comment:
This got missed getting updated when the test switched from negative to positive:
```suggestion
      it 'generates canvas with seeAlso property' do
```